### PR TITLE
Index page of osu-wiki and a minor osu!wiki history update

### PR DESCRIPTION
**Pull request by RaikireHiuduo**
See title.

Updated from #177.

Notes:-
1. Facebook link is not shown in GitHub for some reason.
2. `osu!news` and `osu!dev blog` still uses the old tumblr link. What is the new links for those two?

### CODE REVIEW

File: wiki/Index/en.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- [osu!store](/wiki/osu!store "osu!store")
+- [People](/wiki/People "People")
+
+### Follow
+
+- [Facebook](https://www.facebook.com/osugame "Facebook")
+- [Reddit](https://www.reddit.com/r/osugame/ "r/osugame at Reddit")
+- [Twitter](/wiki/Twitter "Twitter page")
+- [osu!news](http://osunews.tumblr.com/ "osu!news")
+- [osu!dev blog](http://pppy.tumblr.com/ "osu!dev blog")
```
Comment (TPGPL) [inline wiki/Index/en.md]: Link typo

Comment (RaikireHiuduo) [inline wiki/Index/en.md]: To where?

**Comment by RaikireHiuduo** [comment]
@TPGPL So, do you want me to include all the subfolders (if available) too?

File: wiki/Index/en.md (hunk 1 above)
Comment (TPGPL) [inline wiki/Index/en.md]: pppy.tumblr.com/

**Comment by TPGPL** [comment]
I think it would be slightly better, but let's ask other people

**Comment by MegaApplePi** [comment]
@RaikireHiuduo Since this is technically an index page that indexes everything, I suppose it should contain links to everywhere, even if it makes the page appear long.

The links in there will always be in need of maintenance anyways since refactoring and new folders will constantly happen anyway (but redirects may help with half of the battle once they are added :p)

File: wiki/Index/en.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# osu!wiki
+
+Welcome to the osu!wiki, a collaborative compendium of all _osu!_ things!
+
+## Legal notice
```
Comment (MegaApplePi) [inline wiki/Index/en.md]: Title case

File: wiki/Index/en.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+# osu!wiki
```
Comment (MegaApplePi) [inline wiki/Index/en.md]: /me talking (can ignore) _:c the title isn't is same as the folder name_

**Pull request merged**